app/service-catalog/integrations/gitlab.md

```diff
@@ -0,0 +1,58 @@
+---
+title: GitLab
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+works_on:
+    - konnect
+description: The GitLab integration allows you to associate your Service Catalog service to one or more [GitLab projects](https://docs.gitlab.com/ee/user/get_started/get_started_projects.html). 
+
+related_resources:
+  - text: "Service Catalog"
+    url: /service-catalog/
+discovery_support: true
+bindable_entities: "Projects"
+---
+
+For each linked project, the UI can show a **Project Summary** with simple data pulled from the GitLab API, such as the number of open issues, open merge requests, contributors, languages, and latest releases.
+
+## Prerequisites
+
+* You need the [Owner GitLab role](https://docs.gitlab.com/ee/user/permissions.html) to authorize the integration. This is required for event ingestion.
+* Only [GitLab.com subscriptions](https://docs.gitlab.com/ee/subscriptions/gitlab_com/) are supported at this time
```

Comment:
```suggestion
* Only [GitLab.com subscriptions](https://docs.gitlab.com/ee/subscriptions/gitlab_com/) are supported at this time.
```
Since the previous bullet point also includes a period